tests/accessibility-assertions.js

```diff
@@ -37,6 +61,12 @@ const DialogState = {
   InvisibleUnmounted: 'InvisibleUnmounted',
 };
 
+const ListboxState = {
+  Visible: 'Visible',
+  InvisibleHidden: 'InvisibleHidden',
```

Comment:
I don't see use of this state in tests, I assume that we are just lack of some tests right ? Some assertions don't implement it. Do we have a plan to extend the test suite ?

Reply:
Maybe would be nice to follow the same approach what @achambers did in dialog ? What I mean is copy test e.g. from react repo and change to `todo` [example](https://github.com/GavinJoyce/ember-headlessui/blob/master/tests/integration/components/dialog-test.js#L205) 
I am willing to help with them after that and fill gaps.

Reply:
Ah yes this refers to the alternative "hidden" or "unmount" render strategy, in which React will insert the listbox options into the DOM, but display as hidden.

I think it's safe to push this feature out for now.

https://github.com/GavinJoyce/ember-headlessui/pull/76/commits/b452c35fc3e8bb69f110df608dab582f6e148a6d adds `todo`s for these tests.